dagfactory/__main__.py

```diff
@@ -108,5 +111,58 @@ def lint(
         console.print(f"Analysed {len(files)} files, [green]no errors found.[/green]")
 
 
+@app.command()
+def convert(
+    path: Path = typer.Argument(..., help="Path to a YAML file or a directory of YAML files to convert"),
+    # type: str = typer.Option("airflow2to3", "--type", "-t", help="Conversion type (default: airflow2to3)"),
```

Comment:
This PR only support `airflow2to3`? Would it make sense to update the docstring accordingly?